l10n_it_fiscalcode/i18n/it.po

```diff
@@ -1,262 +1,76 @@
 # Translation of Odoo Server.
```

Comment:
Translaiotns must be done by weblate.
See https://odoo-community.org/blog/the-oca-blog-1/post/translations-platform-81